packages/vc-data/src/base/v1.ts

```diff
@@ -10,13 +10,15 @@ export type GovernmentOrgV1 =
   | CreateThing<'GovernmentOrganization'>
   | CreateThing<'AdministrativeArea'>
 
-export type MonetaryAmountRV1 = CreateThing<
+export type MonetaryAmountRV1Mixin = CreateThing<
   'MonetaryAmount',
   {
-    currency: string
-    value: number | string
+    currency?: string
+    value?: number | string
+    availableValue?: number | string
   }
 >
+export type MonetaryAmountRV1 = ExtendThing<MonetaryAmountRV1Mixin, CreateThing<'MonetaryAmount'>>
```

Comment:
Oh I see what's happening here with CI failing. The type this is creating is:

```ts
type MonetaryAmountRV1 = {
  '@type': ['MonetaryAmount', 'MonetaryAmount'],
  // ...
}
```

Since we are adding a field we should follow the pattern we've setup

```ts
type MonetaryAmount = CreateThing<'MonetaryAmount'>

type MonetaryAmountEV1Mixin = CreateThing<'MonetaryAmountE', {
  currency?: string
  value?: number | string
  availableValue?: number | string
}>

// Create entry context for this
export type MonetaryAmountEV1 = ExtendThing<MonetaryAmountEV1Mixin, MonetaryAmount>

// Needs to stay for backwards compatibility
// Create entry context for this
export type MonetaryAmountRV1 = CreateThing<'MonetaryAmount', {
  currency: string
  value: number | string
}>
```

And then in the account credential update `hasValue` to be `hasValue: MonetaryAmountRV1 | MonetaryAmountEV1`